src/components/Header/Navigation/DropdownLink/DropdownLink.tsx

```diff
@@ -2,7 +2,6 @@ import { Playfair_Display, Poppins } from 'next/font/google';
 import styles from './DropdownLink.module.scss';
 import Link from 'next/link';
 import { ReactNode } from 'react';
-import { BsFillCaretDownFill } from "react-icons/bs";
 
 const playfair = Playfair_Display({ subsets: ['latin'] });
```

Comment:
Maybe using a higher font weight will also help?
```suggestion
const playfair = Playfair_Display({ subsets: ['latin'], weight: '500' });
```